support-frontend/assets/pages/showcase/showcase.jsx

```diff
@@ -26,6 +26,15 @@ import ConsentBanner from 'components/consentBanner/consentBanner';
 
 import './showcase.scss';
 import { Provider } from 'react-redux';
+import { sendClickedEvent } from 'helpers/tracking/clickTracking';
+import { trackComponentClick } from 'helpers/tracking/ophan';
+
+// ----- Tracking ----- //
+
+const clickHandler = (ctaType) => {
+  sendClickedEvent(`support_page_cta_${ctaType}`)();
+  trackComponentClick(`support-page-cta-${ctaType}`);
```

Comment:
I think you can just call `trackComponentClick` because this calls GA as well as Ophan. Which means that you don't need this function

Reply:
Great, thanks Rupert!